cmd/gosky/debug.go

```diff
@@ -512,7 +503,7 @@ var debugFeedGenCmd = &cli.Command{
 		}
 
 		if ss == nil {
-			return fmt.Errorf("No '#bsky_fg' service entry found in feedgens DID document")
+			return fmt.Errorf("no '#bsky_fg' service entry found in feedgens DID document")
```

Comment:
Do not capitalize (or punctuate) error strings. See https://go.dev/wiki/CodeReviewComments#error-strings